win32ss/user/ntuser/ntuser.c

```diff
@@ -199,6 +200,11 @@ NtUserInitialize(
     /* Save the EPROCESS of CSRSS */
     InitCsrProcess(/*PsGetCurrentProcess()*/);
 
+    /* Initialize global shared info */
```

Comment:
This gSharedInfo thing is already being done in
https://github.com/HBelusca/reactos/commit/456d1015ab714d78e4ff0ca280a364e74e26857c

Reply:
Awww, indeed! :sweat_smile: Didn't saw your branch earlier @HBelusca. But the problem is, this is not in master yet, although my handle validating implementations are depending on this. So I can't just remove it. And I don't know  when you will decide to finish it and merge in master.